packages/extension/src/manifest.json

```diff
@@ -10,30 +10,38 @@
     "48": "icons/48.png",
     "128": "icons/128.png"
   },
-  "__prod__permissions": [
+  "__prod__host_permissions": [
     "https://daily.dev/",
     "https://*.daily.dev/",
     "https://dailynow.co/",
     "https://*.dailynow.co/"
   ],
-  "__dev__permissions": ["storage", "http://localhost/", "http://*.local.com/"],
-  "optional_permissions": ["topSites", "*://*/*"],
-  "content_security_policy": "script-src 'self' https://www.google-analytics.com; object-src 'self'",
-  "__firefox__content_security_policy": "object-src 'self'",
-  "browser_action": {
+  "__dev__permissions": ["storage"],
+  "__dev__host_permissions": ["http://localhost/", "http://*.local.com/"],
+  "optional_permissions": ["topSites"],
+  "optional_host_permissions": [ "*://*/*"],
+  "content_security_policy": {
```

Comment:
I see we removed the specific definition for firefox, is it on purpose?

Reply:
good catch, since we removed google analytics, there was no meaningful diff between the regular content security policy and the firefox specific one. Was there a particular reason `script-src 'self'` was not initially included there?

We can see during testing in firefox if this causes any problems or triggers a permissions reset.

Reply:
Fair enough, I'm not sure why we added it in the first place but we can leave it for testing

Reply:
having this in also ensures protection agains cross site scripting and other things, so it's a good default to have
 
BTW, I found the answer from daildev search here very nice: https://app.daily.dev/search?id=f69e7f9a-7e23-11ee-9cee-42010a007e0c